snippets/trigger-expressions/trigger-project.update.json

```diff
@@ -1,10 +1,5 @@
 {
```

Comment:
this file should be in the `snippets/trigger-expressions/project-expressions/update.json` path right?

Reply:
Good catch.